### PR TITLE
add complex variant

### PR DESCRIPTION
**Pull request by ltalirz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@
 {% set sha256 = "fe744bd9c92557f0a38027f19a419b6e6765a982f531d02e620c79eb1a97bae7" %}
```
Comment (dalcinl) [inline recipe/meta.yaml:4]: you need `{% set build = 1 %}` in here, right?

**Comment by ltalirz** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (ltalirz) [reply recipe/meta.yaml:5]: indeed, thanks

**Comment by ltalirz** [comment]
Of course one side effect of this is that it doubles the number of builds to 50 ;-)

**Comment by dalcinl** [comment]
> Of course one side effect of this is that it doubles the number of builds to 50 ;-)
> 
> Edit: almost doubles; 25 => 41.
> I don't see a way around this though.

Let it be, let it be... 

There is no way around it, I think.

**Comment by ltalirz** [comment]
@dalcinl CI passes.

The tests currently just import the package. Not sure whether one should add something more there to make sure it actually works

**Pull request merged**